=== modified file 'src/greeter-session.c'
--- src/greeter-session.c	2016-06-29 03:15:44 +0000
+++ src/greeter-session.c	2016-06-29 03:16:05 +0000
@@ -21,10 +21,6 @@
 {
     /* Greeter running inside this session */
     Greeter *greeter;
-
-    /* Communication channels to communicate with */
-    int to_greeter_input;
-    int from_greeter_output;
 };
 
 G_DEFINE_TYPE (GreeterSession, greeter_session, SESSION_TYPE);
@@ -108,8 +104,6 @@
     GreeterSession *self = GREETER_SESSION (object);
 
     g_clear_object (&self->priv->greeter);
-    close (self->priv->to_greeter_input);
-    close (self->priv->from_greeter_output);
 
     G_OBJECT_CLASS (greeter_session_parent_class)->finalize (object);
 }

